vote/vote.go

```diff
@@ -90,17 +91,16 @@ func (v *Vote) Create(ctx context.Context, requestUserID int, r io.Reader) (int,
 	}
 
 	sql := `INSERT INTO poll
-		(title, method, config, visibility, state, content_object_id, meeting_id, result, published, allow_vote_split)
-		VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10)
+		(title, config_id, visibility, state, content_object_id, meeting_id, result, published, allow_vote_split)
+		VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)
 		RETURNING id;`
 
 	var newID int
 	if err := tx.QueryRow(
 		ctx,
 		sql,
 		ci.Title,
-		ci.Method,
-		ci.Config,
+		"poll_config_approval/1", // Temporaty value to make postgres happy. Will be set later.
```

Comment:
@hjanott I have to create the poll first and afterwards add the poll_config. I do this in the same transaction, so this should not be a problem.

but I was not able to do it without this temporary value. Do you have an idea?